packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -86,20 +83,7 @@ export function useGasSummary(): UseGasSummaryResult {
   const [networks] = useNetworks()
   const { childChainProvider, parentChainProvider, isDepositMode } =
     useNetworksRelationship(networks)
-  const [{ amount }] = useArbQueryParams()
-  const debouncedAmount = useDebounce(amount, 300)
-  const decimals = useSelectedTokenDecimals()
-
-  const amountBigNumber = useMemo(() => {
-    if (isNaN(Number(debouncedAmount))) {
-      return constants.Zero
-    }
-    const amountSafe = debouncedAmount || '0'
-
-    const correctDecimalsAmount = truncateExtraDecimals(amountSafe, decimals)
-
-    return utils.parseUnits(correctDecimalsAmount, decimals)
-  }, [debouncedAmount, decimals])
+  const amountBigNumber = useAmountBigNumber()
```

Comment:
We remove debouncing from `useAmountBigNumber`, do we need to add it back here?

Reply:
yea good point, i wanted to align the usage to be identical but it should be debounced here